csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -110,7 +110,7 @@ std::unique_ptr<ReductionParams> getInnerOuterPersistentHeuristics(
                 ref_red_tv, reduced_tv, properties.inner_most_dimension_ndims));
       });
 
-  const auto vectorize_factor = vectorize_helper::getVectorizationFactor(
+  auto vectorize_factor = vectorize_helper::getVectorizationFactor(
```

Comment:
If `vectorize_helper::getVectorizationFactor` is always called, do we need to cache it?